eZ/Bundle/EzPublishCoreBundle/Console/Application.php

```diff
@@ -46,9 +46,10 @@ protected function registerCommands()
         parent::registerCommands();
 
         $container = $this->getKernel()->getContainer();
-        $this->siteAccessName = $this->siteAccessName ?: $container->getParameter( 'ezpublish.siteaccess.default' );
-        $siteAccess = new SiteAccess( $this->siteAccessName, 'cli' );
-        $container->set( 'ezpublish.siteaccess', $siteAccess );
+        $siteAccess = $container->get( 'ezpublish.siteaccess' );
+        $siteAccess->name = $this->siteAccessName ?: $container->getParameter( 'ezpublish.siteaccess.default' );
+        $siteAccess->matchingType = 'cli';
+
```

Comment:
Well, as the service is not synchronized anymore (you can't silence synchronized deprecation warning when set from yaml). I kind of needed a way to update he values of the object, given it is a object I can do it in place and it will be reflected everywhere.

Which makes me think the initial value of SiteAccess should probably reflect that it is not set yet, aka:
```diff
     ezpublish.siteaccess:
          class: %ezpublish.siteaccess.class%
-         arguments: [%ezpublish.siteaccess.default.name%]
+         arguments: [%ezpublish.siteaccess.default.name%, 'uninitialized']
```